unblob/processing.py

```diff
@@ -98,6 +98,10 @@ def _process_task(  # noqa: C901
         log.info("Reached maximum depth, stop further processing")
         return
 
+    if not valid_path(task.path):
+        log.warn("Path contains invalid characters, it won't be processed")
+        return
+
```

Comment:
Important decisions / features like this, should be in a separate commit message which only contains this few lines (and of course the `valid_path` function in `file_utils` and its tests).

But I am okay leaving this with the other changes right now, as the commit message explicitly writes about this decision and its reason.

Reply:
Understood, will keep this in mind !